### ISSUE TITLE
Error when running tests: this version of ember-auto-import requires the app to have a dependency on webpack 

### ISSUE THREAD

**Issue by Gorzas** [minutebase/ember-can#145]
Could anyone try if you have the same problem? I believe it's related to the last release of EmberJS (3.27) when doing `npm run test`:

```
[ember-auto-import] this version of ember-auto-import requires the app to have a dependency on webpack 
```

I found it when [trying to solve this PR](https://github.com/minutebase/ember-can/pull/144#issuecomment-874186819). It seems to have a very easy solution but I need to know I'm not the only one having this issue.

**Comment by patrickberkeley** [comment]
webpack is [required for `ember-auto-import` v2](https://github.com/ef4/ember-auto-import/blob/master/docs/upgrade-guide-2.0.md) so I'm assuming somehow auto import was upgraded. I don't see ember-auto-import v2 as a requirement for ember 3.27, so is there a chance you upgraded it on your own?

**Comment by Gorzas** [comment]
@patrickberkeley I'm going to rm `node_modules` and try again. I also have seen that `package-lock.json` gets updated after doing an `npm install`, don't know if it's related.

**Comment by Gorzas** [comment]
The `ember-auto-import` dependency is directly in the `package.json`: https://github.com/minutebase/ember-can/blob/master/package.json#L68

**Comment by patrickberkeley** [comment]
Ah that would do it :-P Pretty sure a webpack config is required then.

**Comment by Exelord** [comment]
Hmm, all tests are passing for me locally without any issues. Same for CI. What version of node and npm are you using? Im testing on the latest.

**Comment by Gorzas** [comment]
@Exelord Could try to do `rm -rf node_modules` and install the dependencies again, please?

**Comment by Exelord** [comment]
Yes, that's what I did. All works just fine. CI confirms that. :)

**Comment by Gorzas** [comment]
I'm using node 12.22.1 and npm 6.14.12

**Comment by Exelord** [comment]
Try with latest node 14 or 16 and npm 7.19^

**Comment by Gorzas** [comment]
@Exelord It works with Node 14/16 and npm 7.19. I have updated the PR upgrading the requirement of Node version: https://github.com/minutebase/ember-can/pull/146
